src/go/k8s/cmd/configurator/main.go

```diff
@@ -183,7 +183,7 @@ func registerKafkaAPI(c *configuratorConfig, cfg *config.Config) {
 }
 
 func registerAdvertisedKafkaAPI(
-	c *configuratorConfig, cfg *config.Config,
+	c *configuratorConfig, cfg *config.Config, index int,
```

Comment:
nit: should `index int` be a strong type called broker-id
```
type BrokerId int;

```

Reply:
Done